### DISCUSSION TITLE
Default Permission

### DISCUSSION THREAD

**Discussion by derickprado** [netbox-community/netbox#9714] [Q&A] [answered]
Hi everyone,

I'm trying to define a default permission on netbox, since I create a new user, it comes as default, that's possible? 
I tried to set it on a group too, but unsuccessfully.

**Comment by candlerb** [comment, marked as answer]
You can set the permission on a group, and make new users members of that group by default.  This works for remote authentication with [`REMOTE_AUTH_DEFAULT_GROUPS`](https://docs.netbox.dev/en/stable/configuration/remote-authentication/#remote_auth_default_groups). If you are using local authentication, I think you might need to remember to add them to the group when you create the user.

If the objective is simply that all users should get read access to Netbox, or read access to certain object types, then use [`EXEMPT_VIEW_PERMISSIONS`](https://docs.netbox.dev/en/stable/configuration/optional-settings/#exempt_view_permissions`) and [`LOGIN_REQUIRED`](https://docs.netbox.dev/en/stable/configuration/optional-settings/#login_required).

**Reply by derickprado** [reply]
Thank you @candlerb, I inserted '*' on [EXEMPT_VIEW_PERMISSION](https://docs.netbox.dev/en/stable/configuration/optional-settings/#exempt_view_permissions%60) and it worked.
